src/content/docs/usage/installation.md

```diff
@@ -47,31 +48,17 @@ This is important - if not done, dependencies (such as Java) will be installed f
 the wrong channels and things may break in strange ways.
 :::
 
-A best practice with conda is to create an environment and install the tools in it.
-Therefore you will prevent version conflicts and keep everything clean.
-To do so use the following command:
+A best practice with conda is to use a dedicated conda environment.
+This can help to prevent version conflicts and keep everything clean:
 
 ```bash
 conda create --name env_nf nextflow
 conda activate env_nf
 ```
 
-To deactivate the conda environment, run the following command:
-
-```bash
-conda deactivate
-```
-
-> If you're already in the conda environment you want to use, you can just install Nextflow directly:
->
-> ```bash
-> conda install nextflow
-> ```
-
-If you want to develop your own pipelines or collaborate with others using the nf-core guidelines you will need the nf-core packages.
-Please follow the instructions [here](tutorials/nf_core_usage_tutorial.md)
+To deactivate the conda environment, run `conda deactivate`.
```

Comment:
````suggestion
To deactivate the conda environment, run:
```bash
conda deactivate
```
````
to get a copy-code button

Reply:
Fair enough - I went from code block to inline because I was trying to make it shorter but agree the copy code button is nice.